beacon-chain/cache/skip_slot_cache_test.go

```diff
@@ -1,53 +1,34 @@
-package cache_test
```

Comment:
Why revert this?

Reply:
Out of all test files in the cache folder some where in the `cache` package and some in the `cache_test` package so I decided to unify it. I can change it back if you'd like.

Reply:
We should use `pkg_test` whenever we can to mimic block box testing. But sometimes we have to use `pkg` in tests to gain access to the non-exported identifiers. It's for unit tests that require access to non-exported variables, functions, and methods.

`pkg_test` is generally preferred over `pkg` for tests. A good follow up tasks from this is to look at other tests to see if we can convert them to `pkg_test`

Reply:
Furthermore, please, try to keep PRs constrained to the single purpose -- here, you are updating test to use assertions, no further refactoring should make it into this PR. From time to time, our PRs will contain more changes than is warrant by PR's goal, but we are trying to keep them focused, as a general rule.